guides/user/plugins/deploy-example.md

```diff
@@ -1,96 +1,153 @@
+```
+
+The plugin and repository information is a subset of the entries in your Halconfig.
+
+## Redeploy Spinnaker
+
+```bash
+hal deploy apply
+```
+
+## Access the RandomWait plugin in the UI
+
+The RandomWait stage appears in the **Type** select list when you create a new Pipeline stage.
+
+
+
+## Troubleshooting
+
+If the plugin doesn't appear in the **Type** select list, check the following logs:
+
+* Orca, for the plugin backend
+
+  You should see output similar to this when the plugin has been successfully loaded:
+
+  ```bash
+  2020-07-02 16:12:43.284  INFO 1 --- [           main] com.netflix.spinnaker.orca.Main          : [] Starting Main v1.0.0 on spin-orca-7466444f64-cg5gd with PID 1 (/opt/orca/lib/orca-web.jar started by spinnaker in /)
+  2020-07-02 16:12:54.691  INFO 1 --- [           main] org.pf4j.DefaultPluginManager            : [] PF4J version 3.2.0 in 'deployment' mode
+  2020-07-02 16:12:59.088  INFO 1 --- [           main] .k.p.u.r.s.LatestPluginInfoReleaseSource : [] Latest release version '1.1.14' for plugin 'Armory.RandomWaitPlugin'
+  2020-07-02 16:12:59.091  INFO 1 --- [           main] .k.p.u.r.s.SpringPluginInfoReleaseSource : [] Spring configured release version '1.1.14' for plugin 'Armory.RandomWaitPlugin'
+  2020-07-02 16:12:59.103  INFO 1 --- [           main] p.u.r.s.PreferredPluginInfoReleaseSource : [] No preferred release version found for 'Armory.RandomWaitPlugin'
+  2020-07-02 16:12:59.620  INFO 1 --- [           main] org.pf4j.util.FileUtils                  : [] Expanded plugin zip 'orca.zip' in 'orca'
+  2020-07-02 16:12:59.643  INFO 1 --- [           main] org.pf4j.util.FileUtils                  : [] Expanded plugin zip 'Armory.RandomWaitPlugin-pf4jStagePlugin-v1.1.14.zip' in 'Armory.RandomWaitPlugin-pf4jStagePlugin-v1.1.14'
+  2020-07-02 16:12:59.652  INFO 1 --- [           main] org.pf4j.util.FileUtils                  : [] Expanded plugin zip 'orca.zip' in 'orca'
+  2020-07-02 16:12:59.653  INFO 1 --- [           main] org.pf4j.AbstractPluginManager           : [] Plugin 'Armory.RandomWaitPlugin@unspecified' resolved
+  2020-07-02 16:12:59.658  INFO 1 --- [           main] org.pf4j.AbstractPluginManager           : [] Start plugin 'Armory.RandomWaitPlugin@unspecified'
+  2020-07-02 16:12:59.659  INFO 1 --- [           main] i.a.p.s.wait.random.RandomWaitPlugin     : [] RandomWaitPlugin.start()
+  ```
+
+  If you see log output similar to
+
+  ```bash
+  Plugin 'Armory.RandomWaitPlugin@unspecified' requires a minimum system version of orca>=8.0.0, and you have 1.0.0
+  2020-07-01 16:52:13.170  WARN 1 --- [           main] org.pf4j.AbstractPluginManager           : [] Plugin '/opt/orca/plugins/Armory.RandomWaitPlugin-pf4jStagePlugin-v1.1.13/orca' is invalid and it will be disabled
+  ```
+
+  your plugin doesn't work with the version of Spinnaker you are using. Contact the plugin's developer.
```

Comment:
Because this just continues the previous sentence, maybe start with "..."

Reply:
done